### ISSUE TITLE
Callbacks Fail to Execute on Android OS

### ISSUE THREAD

**Issue by manifestinteractive** [uxcam/cordova-uxcam#5]
#### Overview:

Following the documentation to start and stop UXCam in Cordova works perfectly fine for iOS devices, however, on two different Android Devices ( both Nexus 5's ) the exact same code with callbacks failed to be triggered.
#### Acceptance Criteria:

`successCallback` and `errorCallback` should work for Android Devices on all UXCam methods with callbacks.
#### Steps to Duplicate:

Here is an example of the code expected to work ( replacing `my-api-key` with the actual API key, of course ):

``` js
UXCam.startWithKey('my-api-key', function () { console.log('start success') }, function () { console.log('start error') })
```

Expected output from that command should be a console log with either `start success` or `start error`.  But currently running this on an Android device you get neither.

Same goes for the following:

``` js
UXCam.stopApplicationAndUploadData(function () { console.log('stop success') }, function () { console.log('stop error') })
```

Expected output from that command should be a console log with either `stop success` or `stop error`.  But again running this on an Android device you get neither.
#### System Info:

Android OS v6.0.1 ( Tested on two different Nexus 5's with failure )
#### Relevant Documentation

I am referencing the `successCallback` and `errorCallback` parameters in the UXCam methods in this file:

https://github.com/uxcam/cordova-uxcam/blob/master/www/UXCam.js
#### My feelings on this Issue

![sad_baby](https://cloud.githubusercontent.com/assets/508411/18226451/665544d2-71d8-11e6-91dd-287f31a77ea9.gif)

**Comment by uxcam-android** [comment]
I have pushed fix on branch v2.3.3. Could you please verify? @manifestinteractive 
https://github.com/uxcam/cordova-uxcam/tree/v2.3.3

**Comment by manifestinteractive** [comment]
Verified that v2.3.3 resolves both Android issues I was having.  Installed plugin via: ( for others that might want to use this branch )

```
cordova plugin add https://github.com/uxcam/cordova-uxcam.git#v2.3.3
```
